Local Pods/GitHubAPI/GitHubAPI/V3Notification.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  V3Notification.swift
+//  GitHubAPI
+//
+//  Created by Ryan Nystrom on 3/3/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+public struct V3Notification: Codable {
+
+    enum Reason: String, Codable {
+        case assign = "assign"
+        case author = "author"
+        case comment = "comment"
+        case invitation = "invitation"
+        case manual = "manual"
+        case mention = "mention"
+        case stateChange = "state_change"
+        case subscribed = "subscribed"
+        case teamMention = "team_mention"
+        case reviewRequested = "review_requested"
+    }
```

Comment:
Why not like this?

```
enum Reason: String, Codable {
  case assign
  case author
  case comment
  case invitation
  case manual
  case mention
  case stateChange
  case subscribed
  case teamMention = "team_mention"
  case reviewRequested = "review_requested"
}
```

Reply:
Used a script to gen these, but ya same thing 😄